source/anatomy.cpp

```diff
@@ -0,0 +1,39 @@
+// Copyright (c) 2011-2024 The Khronos Group, Inc.
```

Comment:
All the other examples have a copyright notice like this:

```
// SPDX-FileCopyrightText: 2024 The Khronos Group Inc.
```

I think we should be consistent with how we mark the copyright notice.  Either use the SPDX form everywhere or change them all to the more traditional form.